todoapp.js

```diff
@@ -0,0 +1,19 @@
+document.addEventListener("DOMContentLoaded", () => {
+  let sub = document.querySelector("#submit");
+  let list = document.getElementById("list");
```

Comment:
Currently you do not have an element with the id `list`, so you could change this line to 
```
let list = document.querySelector(".list")
```